packages/utilities/src/a11yModalize.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * The helper functions here will make the target element as modal to screen readers, by placing aria-hidden on elements
+ * that are siblings to the target element and the target element's ancestors (because aria-hidden gets inherited).
+ * That way, all other elements on the page are hidden to the screen reader.
+ */
+
+/** Call this on a target element to make it modal to screen readers.
+ * This returns an array of elements, keep that array and pass it to removeModalize() when the target is no longer modal
+ * or no longer on the page.
+ */
+export function modalize(target: HTMLElement): HTMLElement[] {
+  const affectedNodes: HTMLElement[] = [];
+
+  // start at target, then recurse and do the same for parent, until we reach <body>
+  while (target !== document.body) {
```

Comment:
call `getDocument(target)` to get the correct document. When rendering in child windows, the document global is the host rather than the child window.